### PR TITLE
Feature editable install

### PR DESCRIPTION
**Pull request by FFroehlich**

### CODE REVIEW

**Comment by FFroehlich** [comment]
To test this, run `pip3 install -e git+https://github.com/ICB-DCM/AMICI@feature_editable_install#egg=amici\&subdirectory=python/sdist`

**Comment by dweindl** [comment]
> To test this, run `pip3 install -e git+https://github.com/ICB-DCM/AMICI@feature_editable_install#egg=amici\&subdirectory=python/sdist`

Took only 9min to complete :sweat_smile:

File: python/amici/setuptools.py [hunk 1]
```diff
@@ -23,12 +23,8 @@ def getBlasConfig():
     """
 
     if pkgconfig:
-        blaspkgcfg = pkgconfig.parse('cblas')
-        # NOTE: Cannot use pkgconfig.exists('cblas'), since this is true
-        # although no libraries or include dirs are available
-        blaspkgcfg['found'] = 'include_dirs' in blaspkgcfg \
-                              and blaspkgcfg['include_dirs']
-        if blaspkgcfg['found']:
+        if pkgconfig.exists('cblas'):
```
[the remark this answers is not in the document]
Comment (FFroehlich) [reply python/amici/setuptools.py]: I would assume that for your case pkgconfig did find cblas, but include_dirs were empty as the include path was already contained in the system include path and pkgconfig deduplicated the include path from cblas.pc?

[the remark this answers is not in the document]
Comment (dweindl) [reply python/amici/setuptools.py]: I think it was not for blas exactly, but for hdf5 in windows, on a system where it was not installed. This made me lose my trust in .exists().

File: python/amici/setuptools.py [hunk 2]
```diff
@@ -136,17 +138,55 @@ def addDebugFlagsIfRequired(cxx_flags, linker_flags):
...
         print("ENABLE_AMICI_DEBUGGING was set to TRUE."
               " Building AMICI with debug symbols.")
         cxx_flags.extend(['-g', '-O0'])
         linker_flags.extend(['-g'])
+
+
+def generateSwigInterfaceFiles():
+    """Compile the swig python interface to amici
+    """
+    swig_outdir = '%s/amici' % os.path.abspath(os.getcwd())
```
[the remark this answers is not in the document]
Comment (FFroehlich) [reply python/amici/setuptools.py]: cwd does not change based on where the script is located, right?

[the remark this answers is not in the document]
Comment (dweindl) [reply python/amici/setuptools.py]: Right :-/

**Pull request merged**